.github/workflows/workflow.yml

```diff
@@ -0,0 +1,15 @@
+name: Lighthouse Audit
+on: [push, pull_request]
```

Comment:
```suggestion
on: [pull_request]
```

Reply:
push doesn't seem to work. does `pull_request` rerun when the branch updates though?